src/SubjectHead/style.js

```diff
@@ -0,0 +1,119 @@
+import { makeStyles } from '@material-ui/core/styles';
+import colors from '../styles/colors';
+
+const useStyles = makeStyles({
+    body:{ 
+        fontSize:"62.5%", 
+    },
+    container: {
```

Comment:
avoid using generic names like `container`, you can have something specific to component like `subjectHeadContainer`